quarkus/admin/src/main/java/org/apache/polaris/admintool/PurgeCommand.java

```diff
@@ -37,9 +38,26 @@ public class PurgeCommand extends BaseCommand {
   @Override
   public Integer call() {
     try {
-      metaStoreManagerFactory.purgeRealms(realms);
-      spec.commandLine().getOut().println("Purge completed successfully.");
-      return 0;
+      var result = metaStoreManagerFactory.purgeRealms(realms);
+      var failed =
+          result.entrySet().stream()
+              .filter(e -> !e.getValue().isSuccess())
+              .map(Map.Entry::getKey)
+              .toList();
+      if (failed.isEmpty()) {
+        spec.commandLine().getOut().println("Purge completed successfully.");
+        return 0;
+      }
+
+      var out = spec.commandLine().getOut();
```

Comment:
This should be inlined or pulled above line 47

Reply:
Rather a nit, right?